subscriptions/subscriptions-impl/src/main/java/com/duckduckgo/subscriptions/impl/pixels/SubscriptionPixel.kt

```diff
@@ -259,9 +259,18 @@ enum class SubscriptionPixel(
         types.associateWith { type -> if (withSuffix) "${baseName}_${type.pixelNameSuffix}" else baseName }
 }
 
+object SubscriptionPixelParameter {
+    const val ERROR_TYPE = "errorType"
```

Comment:
I'd add another parameter called "reason" or similar to separate it from the error type and make our life a bit easier.